webapp/src/jaas/components/conference/jaas_conference.tsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import {id as pluginId} from '../../../manifest';
+import {AnyAction} from 'redux';
+import {StartMeetingWindowAction, startMeetingWindowActionCreator} from '../../actions';
+import {ThunkDispatch} from 'redux-thunk';
+
+type Props = {
+    actions: {
+        startJaaSMeetingWindow: (jwt: string | null, path: string | null) => Promise<StartMeetingWindowAction>
+    }
+}
+
+type State = {
+    jaasJwt?: string | null,
+    jaasRoom?: string | null,
+}
+
+export class JaaSConference extends React.PureComponent<Props, State> {
```

Comment:
Do we need to convert this component to functional or not

Reply:
There is problem while converting it to functional component. So, for now leaving it as class component.